### PR TITLE
Update index.html -- add changes for review

### PR DESCRIPTION
**Pull request by webd-student1**
Please review my changes....

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -50,6 +50,9 @@ <h1>Hello and welcome to my site!</h1>
         <p>This is filler text, otherwise known as lorem ipsum. It is nonsense text to fill space. There are generators of funny lorem ipsum, such as the one from <a href="https://montypythonipsum.megert.se/">Monty Python</a> or <a href="http://fillerama.io/">others</a>. Oh, I don't have time for this. I have to go and buy a single piece of fruit with a coupon and then return it, making people wait behind me while I complain.</p>
         <h3>Week 2</h3>
         <p>This is filler content for week 2. </p>    
+
+        <h3>Please Review</h3>
+        <p>Submitting this changed file for review....</p>
```
Comment (dselicaro) [inline index.html]: This looks good but you need much more...

**Review by dselicaro — CHANGES_REQUESTED** [review]
Hi Doggo -- I reviewed your code and I found some other changes. 
Thanks,
Dan

**Comment by webd-student1** [comment]
Hi Professor -- I have made the required changes!

**Comment by dselicaro** [comment]
Looks good but try again. Add some more lorem ipsum please. Thank you!

**Pull request closed without merging**